packages/editor-ui/src/components/NodeCreator/NodeItem.vue

```diff
@@ -0,0 +1,84 @@
+<template functional>
+	<div :class="{[$style['node-item']]: true, [$style.bordered]: props.bordered}">
+		<NodeIcon :class="$style['node-icon']" :nodeType="props.nodeType" :style="{color: props.nodeType.defaults.color}" />
+		<div>
+			<div :class="$style.details">
+				<span :class="$style.name">{{props.nodeType.displayName}}</span>
+				<span :class="$style['trigger-icon']">
+					<TriggerIcon v-if="$options.isTrigger(props.nodeType)" />
+				</span>
+			</div>
+			<div :class="$style.description">
+				{{props.nodeType.description}}
+			</div>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+
+import Vue from 'vue';
+import { INodeTypeDescription } from 'n8n-workflow';
+
+import NodeIcon from '../NodeIcon.vue';
+import TriggerIcon from '../TriggerIcon.vue';
+
+Vue.component('NodeIcon', NodeIcon);
+Vue.component('TriggerIcon', TriggerIcon);
+
+export default {
+	props: [
+		'active',
+		'filter',
+		'nodeType',
+		'bordered',
+	],
+	isTrigger (nodeType: INodeTypeDescription): boolean {
+		return nodeType.group.includes('trigger');
+	},
+};
+</script>
+
+<style lang="scss" module>
```

Comment:
module scss works better than scoped scss with functional component. 
it's also theoretically faster for browser to handle. 
avoids scoping issues where parent and child meet.